met-api/src/met_api/schemas/survey.py

```diff
@@ -24,3 +24,10 @@ class Meta:  # pylint: disable=too-few-public-methods
     updated_date = fields.Str(data_key='updated_date')
     engagement_id = fields.Str(data_key='engagement_id')
     engagement = fields.Nested(EngagementSchema)
+    comments_count = fields.Method('get_comments_count')
```

Comment:
do we need this here?
all the time when the get on survey is invoked , it will have to do the count ;which in turn will have to join comments table...
i am okay to keep it as well..but worried about the unnecessary fetch

Reply:
In the design comments count is needed for each survey in the survey listing. https://xd.adobe.com/view/ae75b5d1-8424-4323-be5a-8bf2fe0112f3-fb26/screen/2a77ed82-38d7-4a24-be8a-f6d329a807ed/. What is a better way to get comments count?